waspc/src/Wasp/Generator/WebAppGenerator/RouterGenerator.hs

```diff
@@ -78,11 +84,14 @@ createRouterTemplateData spec =
   RouterTemplateData
     { _routes = routes,
       _pagesToImport = pages,
-      _isAuthEnabled = isAuthEnabled spec
+      _isAuthEnabled = isAuthEnabled spec,
+      _isExternalAuthEnabled = (AS.App.Auth.isExternalAuthEnabled <$> maybeAuth) == Just True,
+      _isGoogleAuthEnabled = (AS.App.Auth.isGoogleAuthEnabled <$> maybeAuth) == Just True
     }
   where
     routes = map (createRouteTemplateData spec) $ AS.getRoutes spec
     pages = map createPageTemplateData $ AS.getPages spec
+    maybeAuth = AS.App.auth $ snd $ getApp spec
```

Comment:
This line appears in many different places. Any ideas on how to reduce duplication?

Reply:
Can think on it. Not sure it justifies a common helper given how small it is though, but maybe it does. 🤔

Reply:
I'd probably add it, but your call.

Reply:
I decided to punt on this one for now, just to avoid touching many more files where this is used.